File: Android/app/src/main/java/com/accelo/data/model/Activity.kt
package com.accelo.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


/**
 * Created by Yuri Popiv on 11/25/2019.
 */
data class Activity(
    @SerializedName("preview_body")
    @Expose
    var previewBody: String? = null,
    @SerializedName("date_logged")
    @Expose
    var dateLogged: String? = null,
    @SerializedName("confidential")
    @Expose
    var confidential: Int? = null,
    @SerializedName("subject")
    @Expose
    var subject: String? = null,
    @SerializedName("interacts")
    @Expose
    var interacts: List<Owner>? = null,
    @SerializedName("id")
    @Expose
    var id: String? = null,
    @SerializedName("event_text")
    @Expose
    var eventText: String? = null

)

File: iOS/Accelo/Model/Activity.m
//
//  Activity.m
//  Accelo
//
//  Created by Kuryliak Maksym on 06.12.2019.
//  Copyright © 2019 Wise-Engineering. All rights reserved.
//

#import "Activity.h"

@implementation Activity

+ (NSDictionary *)JSONKeyPathsByPropertyKey {
    return @{
        @"confidential": @"confidential",
        @"previewBody": @"preview_body",
        @"subject": @"subject",
        @"dateLogged": @"date_logged",
        @"interacts": @"interacts",
        @"activityID": @"id",
        @"eventText": @"event_text"
    };
}

+ (NSValueTransformer *)interactsJSONTransformer {
    return [MTLJSONAdapter arrayTransformerWithModelClass:[Interact class]];
}

@end
